packages/react-error-overlay/package.json

```diff
@@ -42,28 +42,28 @@
     "babel-loader": "8.0.6",
     "babel-preset-react-app": "^9.1.0",
     "chalk": "3.0.0",
-    "chokidar": "^2.1.2",
-    "cross-env": "5.2.1",
+    "chokidar": "3.3.0",
+    "cross-env": "6.0.3",
     "eslint": "^6.1.0",
-    "eslint-config-react-app": "^5.1.0",
-    "eslint-plugin-flowtype": "3.13.0",
-    "eslint-plugin-import": "2.18.2",
+    "eslint-config-react-app": "5.1.0",
```

Comment:
We'll want to revert this unpinning I think

Reply:
✔️